app/views/posts/_friends.html.erb

```diff
@@ -1,20 +1,15 @@
 <div class="col friends-col py-4">
 	<p class="lead font-red">Friends</p>
-	<p class="d-flex user-name-and-image">
-		<span class="post-author-img">
-		<img src="https://avatars1.githubusercontent.com/u/25789605?s=460&v=4" alt="small profile image" class="img-fluid small-profile-img">
-	</span> 
-	<span class="post-author-name ml-2">Asha Asha</span></p>
-
-	<p class="d-flex user-name-and-image">
-		<span class="post-author-img">
-		<img src="https://avatars1.githubusercontent.com/u/25789605?s=460&v=4" alt="small profile image" class="img-fluid small-profile-img">
-	</span> 
-	<span class="post-author-name ml-2">Asha Asha</span></p>
-
-	<p class="d-flex user-name-and-image">
-		<span class="post-author-img">
-		<img src="https://avatars1.githubusercontent.com/u/25789605?s=460&v=4" alt="small profile image" class="img-fluid small-profile-img">
-	</span> 
-	<span class="post-author-name ml-2">Asha Asha</span></p>
+	<% current_user.friendships.where(confirmed:"approved").each do |friendship| %>
```

Comment:
Never put logic in your views. Consider making this a scope in the user model, then use it in your controller to initialize an instance variable (eg: `@approved_friends = current_user.approved_friends`)

NB: Do not forget to read the article I made reference to above.